### ISSUE TITLE
Default ClusterRoles for RBAC give error on rio ps -a

### ISSUE THREAD

**Issue by rancher-max** [rancher/rio#630]
**Describe the bug**
When running `rio ps -a` while impersonating rio-standard, rio-readonly, or rio-privileged there is a panic error.

**To Reproduce**
1. `kubectl apply -f crb.yaml`
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: rio-standard-crb
subjects:
- kind: Group
  name: rio:standard
  apiGroup: rbac.authorization.k8s.io
roleRef:
  kind: ClusterRole #this must be Role or ClusterRole
  name: rio-standard # this must match the name of the Role or ClusterRole you wish to bind to
  apiGroup: rbac.authorization.k8s.io
```
2. Update kubeconfig to use the rio-standard clusterrole:
```
...
users:
- name: default
  user:
    password: fakepass
    username: fakeuser
    as: rio-standard
    as-groups:
    - rio:standard
```
3. Run `rio ps -a`

**Expected behavior**
Should show available routers and externalservices as well as services.

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-19T13:57:45Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2-k3s.1", GitCommit:"b8b17ba55f20e590df507fce333dfee13ab438c6", GitTreeState:"clean", BuildDate:"2019-10-16T05:17Z", GoVersion:"go1.13.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Rio version**: `v0.6.0-alpha.1`

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2.

**Comment by rancher-max** [comment]
Working in v0.6.0-alpha.2
